filebeat/CHANGELOG.md

```diff
@@ -28,32 +28,26 @@
 
 * Remove the first slash from the path to the service_checks.json file in each integration's manifest.json file
 
-## 0.3.0 / Unreleased
-
-### Add service check
+## 0.3.0
 
 ***Added***: 
 
 * Add [Service check](https://docs.datadoghq.com/developers/service_checks/agent_service_checks_submission/)
 
-
 ## 0.2.0 / Unreleased
 
-### Making the filebeat check compatible with versions > 5.x, and making it able to use the HTTP profiler
-
 ***Added***: 
 
+* Making the filebeat check compatible with versions > 5.x, and making it able to use the HTTP profiler
+
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/DataDog/integrations-extras/pull/2050